tests/timeslots_test.php

```diff
@@ -154,10 +154,17 @@ public function test_valid_crud () {
         $this->assertEquals($thistimeslot->observee_id, $event->userid);
 
         // Test delete.
+        $this->preventResetByRollback();
```

Comment:
I think this isnt needed, and perhaps should just be $this->resetAfterTest(true) at the top